.gitignore

```diff
@@ -82,3 +82,4 @@ package-lock.json
 
 #tmp folder
 */.tmp
+celerybeat-schedule
```

Comment:
?

Reply:
it's generated by celery on startup, didn't think we should check it in. Doesn't happen for you?

Reply:
I don't know that I've run it with the full celery setup still, so did not notice. That's the type of thing that should be in a gitignore though 👍